### PR TITLE
Upgrade c2cgeoform

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: admin/c2cgeoportal_admin/templates/layout.jinja2 [hunk 1]
```diff
@@ -32,16 +32,25 @@
...
+    </script>
+    {% endfor %}
+
     <!-- Deform JavaScript -->
     {% if deform_dependencies %}
-    <script type="text/javascript" src="{{request.static_url('deform:static/scripts/deform.js')}}" type="text/css" />
+    <script src="{{request.static_url('deform:static/scripts/deform.js')}}" type="text/css" />
     {% for js in deform_dependencies['js'] %}
-    <script type="text/javascript" src="{{request.static_url(js)}}"></script>
+    <script src="{{request.static_url(js)}}"></script>
```
Comment (sbrunner) [inline admin/c2cgeoportal_admin/templates/layout.jinja2]: Why this? is it deprecated?

File: commons/c2cgeoportal_commons/models/main.py [hunk 2]
```diff
@@ -67,6 +71,13 @@ def __init__(self, *args: Any, **kwargs: Any):
 _schema: str = config["schema"] or "main"
 _srid: int = cast(int, config["srid"]) or 3857
 
+# Set some default values for the admin interface
+_admin_config: Dict = config.get_config().get("admin_interface", {})
+_map_config: Dict = {**_admin_config.get("map", {}), **default_map_settings}
```
Comment (sbrunner) [inline commons/c2cgeoportal_commons/models/main.py]: default_map_settings should be first
```
>>> {**{'dd': 2, 'ee': 3 }, **{'dd': 4, 'gg': 5}}
{'dd': 4, 'ee': 3, 'gg': 5}`
```

File: geoportal/c2cgeoportal_geoportal/__init__.py [hunk 3]
```diff
@@ -128,7 +130,17 @@ def add_admin_interface(config):
         )
         config.add_view(c2cgeoportal_geoportal.views.add_ending_slash, route_name="admin_add_ending_slash")
         config.add_route("admin_add_ending_slash", "/admin", request_method="GET")
-        config.include("c2cgeoportal_admin", route_prefix="/admin")
+        config.include("c2cgeoportal_admin")
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/__init__.py]: This is no more working with get it fixed?

File: admin/c2cgeoportal_admin/templates/layout.jinja2 (hunk 1 above)
Comment (arnaud-morvan) [reply admin/c2cgeoportal_admin/templates/layout.jinja2]: It is the default value (see with @tonio )

File: commons/c2cgeoportal_commons/models/main.py (hunk 2 above)
Comment (arnaud-morvan) [reply commons/c2cgeoportal_commons/models/main.py]: good catch, thanks

File: admin/c2cgeoportal_admin/templates/layout.jinja2 (hunk 1 above)
Comment (tonio) [reply admin/c2cgeoportal_admin/templates/layout.jinja2]: https://developer.mozilla.org/fr/docs/Web/HTML/Element/script#attr-type

File: geoportal/c2cgeoportal_geoportal/__init__.py (hunk 3 above)
Comment (arnaud-morvan) [reply geoportal/c2cgeoportal_geoportal/__init__.py]: We made changes in c2cgeoform to support multi applications.
The routes now looks like : 
c2cgeoform_index: /{application}/{table}
So the admin app segment is not a route_prefix anymore, it is an option of the application registered in c2cgeoportal_admin.routes module.

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/__init__.py]: OK, thanks :-)

**Comment by sbrunner** [comment]
:-)

**Pull request merged**